scripts/check_golang_version.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# shellcheck disable=2072
+
```

Comment:
A comment here explaining the purpose of this script would be good...indicate the default case, and why it would be run with 'dev'.

Reply:
I'll add comments.